utils/time.go

```diff
@@ -35,3 +38,28 @@ func ParseTime(s string) time.Time {
 	}
 	return time
 }
+
+func NewRandomizedTicker(interval time.Duration, randomDelta time.Duration) chan time.Time {
+	deltaIntervalMs := int(randomDelta.Milliseconds())
+	ch := make(chan time.Time, 1)
+	ticker := newRandomTicker(interval, deltaIntervalMs)
+	go func() {
+		for {
+			now := <-ticker.C
```

Comment:
Rather than repeatedly re-creating a ticker object would it not be simpler to just call `time.Sleep()` here?